hcipy/aperture/__init__.py

```diff
@@ -15,10 +15,12 @@
 	'make_gmt_aperture',
 	'make_tmt_aperture',
 	'make_hexagonal_segmented_aperture',
+    'make_hexagonal_segmented_aperture',
```

Comment:
Why double this line?

Reply:
That happened during the rebasing. I removed it.